jobs/integr8ly/installation-pipeline-qe-pony.yaml

```diff
@@ -63,7 +63,13 @@
 
                 stage('Clone the installer') {
                     dir('installation') {
-                        git branch: BRANCH, url: REPOSITORY
+                        checkout([
+                            $class: 'GitSCM',
+                            branches: [[name: BRANCH]],,
+                            doGenerateSubmoduleConfigurations: false,
```

Comment:
@tremes `doGenerateSubmoduleConfigurations` and `extensions` parameters seem to be repeating and providing no value (empty array and false). Would it be possible to remove those?

Reply:
Oh yes, it can be likely removed.

Reply:
I have actually read about the `doGenerateSubmoduleConfigurations` and sometimes without setting it to false the build can fail, so we might have it here. Any thoughts @trepel ?

Reply:
No thoughts @pawelpaszki . If so, let's have it there. I encountered this issue yesterday as well so I guess we should merge and update the jobs sooner rather than later.

Reply:
Sure. lets merge it. @tremes do you want to update the job definitions?